### ISSUE TITLE
Create todo list generic container

### ISSUE THREAD

**Issue by JSerwatka** [JSerwatka/things-i-should-do#6]
There are 2 types of todo lists:
* daily
* custom

### Daily todo
This todo type needs to have:
* date and week day as title + current day needs to have different styling
* ability to postpone not done tasks

### Custom todo
This todo type needs to have:
* title that can be changed (maybe ability to add emojis and change color)
* button to delete a list
* ability to move order of lists

### Generic todo
Generic todo needs to implement common functionality of both those lists and add simple interface to transform it to daily/custom todo

Blocked by: 
#3
#5
